### ISSUE TITLE
The Modal's close button renders off screen in Internet Explorer and Edge

### ISSUE THREAD

**Issue by ryanewtaylor** [Semantic-Org/Semantic-UI-React#3205]
## Bug Report

The Modal's close button renders off screen in Internet Explorer and Edge. It displays fine in Chrome and Firefox.

### Steps
1. Create a modal with closeButton enabled on it. 
2. Use IE or Edge to render the page

### Expected Result

The close button should display.

### Actual Result

The close button is rendered off screen.

### Version

Seen in

    "semantic-ui-css": "^2.3.2",
    "semantic-ui-react": "^0.81.3",

Not seen in

    "semantic-ui-css": "^2.2.12",
    "semantic-ui-react": "^0.77.0",

### Testcase
https://codesandbox.io/s/k91xxj9zno

The page fails to load at all in IE but the missing close button can be seen in Edge.

**Comment by layershifter** [comment]
Please make sure that you're using matching versions of SUI and SUIR. I checked your codesandbox with SUI 2.4 and SUIR 0.82.5 and issue is not reproduced.

https://codesandbox.io/s/3vnnqlvn1m

**Comment by ryanewtaylor** [comment]
@layershifter how might I find the matching SUI for any given SUIR. I am actually running 0.83.1 in production (my bad on the code sample). The SUIR changelog indicates that 0.83.x was updated to SUI 2.3. However, when I use that combination the close button remains broken in IE and Edge.

**Comment by layershifter** [comment]
> how might I find the matching SUI for any given SUIR.

This information is mentioned in docs: http://react.semantic-ui.com/usage#css

> I am actually running 0.83.1 in production (my bad on the code sample). The SUIR changelog indicates that 0.83.x was updated to SUI 2.3.

0.83.0 was not released yet, the latest version is 0.82.5.

I've checked your example with SUI 2.3 and bug was reproduced, so there is only one way to fix it - update SUI to 2.4. It's sad, but we can't fix CSS bugs from our side.

```
"semantic-ui-css": "^2.4.0",
"semantic-ui-react": "^0.82.5",
```

These deps are correct and will resolve your issue.

The current version of SUIR is fully compatible with SUI 2.4, but doesn't contain introduced features. The next release of SUIR will include them, #3162.
